sdk/src/main/aidl/com/robotemi/sdk/ISdkService.aidl

```diff
@@ -361,4 +362,8 @@ interface ISdkService {
     String startMeeting(in String packageName, in List<Participant> participants, boolean firstParticipantJoinedAsHost);
 
     int configMinimumObstacleDistance(in String packageName, int value);
+
+    List<TourModel> getAllTours(in String packageName);
+
+    void playTour(in String packageName, in String tourId);
```

Comment:
I would prefere to return int or string, to notify permission error, tour doesn't exist error, etc.